spec/builder/input_spec.rb

```diff
@@ -72,7 +72,7 @@
         end
 
         it 'should set and "optional" class also when there is presence validator' do
-          @new_post.class.should_receive(:validators_on).with(:title).any_number_of_times.and_return([
+          @new_post.class.stub(:validators_on).with(:title).and_return([
```

Comment:
Rspec was complaining that should_receive...any_number_of_times was deprecated and should be replaced with stub.  
Not exactly authoritative, but here's a good explanation.  any_number_of_times includes zero, so the should_receive is basically short circuited and functioning like a stub.
http://stackoverflow.com/questions/19227124/alternate-of-any-number-of-times-method-in-stubs